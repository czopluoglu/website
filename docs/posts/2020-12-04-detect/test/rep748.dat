01110101001101110111
11001010000001000111
01000000001110010111
01000001011010111111
11100110001000101110
00000001110001000110
10110101011111111011
10111111110111100110
11011001110010101010
00111001101111111111
10001101100111101111
01000100101010001010
00010101000000010010
00101000000010100000
10011111111101000011
00000010010000101101
00111000110000001100
11111011111111011110
00010000001000101010
10000001000010011100
10100111111110000110
10110100111001001111
11010111110000001010
00001000001001000000
10111111101110110110
00010001001010100111
10101000000111111010
11000101010000111110
01111001111111001010
10100010100110110011
11011001111101101110
00010100111011011110
00111101111110100001
10101101111111101111
01001001111110111010
11100101111110111111
10101010011011111111
01100111011011111111
10011000111001000010
11011101011100001010
11100000001011001101
11010100111111101111
01100000011000001101
10011000011111001111
01001001001000001111
11110111111101111111
10100101000010000010
11110011111000101010
10100001101001101010
10111111000010100110
11000001001011101111
01011101101010010110
00001111101110110001
11101111111100110111
11011011110011000001
11000100100000100011
11010010100111010101
01001000000001100011
11110000111100010111
00101000101010000111
10110000011100111111
10011111011100010011
11111011110110111011
11100101101110100010
10000000011001101001
11000011000010100100
01001001001011000100
11110111101011001111
11110110111000101011
11000001101011100011
01001001000000000010
11001001110101101111
11101000111111000111
11100011111011001010
10000011001000001000
00001100001001100111
01000001001010110111
01000000011111011101
11001010111011111111
11100100111100001010
01110100011000010111
11111101111101111111
10100101111001110100
10111000101010111111
00001000011011001100
10010011101101111111
01011001100001100100
10000111111110111111
10110111011001101111
01001111101010101111
00010001101111101110
11001010001100101011
11100011101011000111
00100101101010111100
10100111111010101111
00101100001101110011
00001101010010000111
01001110011010010100
11101110111110000011
11111111101001100111
11000101100000000011
01111111000010110111
11110111111000001101
11101100011110010111
11011100010100110110
11000110000101110110
11111111101001010111
10000100001100110010
01101001110011100111
11110111011110100010
00100110000000000111
10101001011110111011
10111011111001100100
01110100011011111111
11111100101101101111
00010110101110111110
00001011010110100010
00001001000000110100
10100100001111101111
11111011111111111101
00101101011001111111
00010000001010100111
11010111111111011111
01010100011110110100
01001001000100000000
11101100011011011110
10111101111000101110
00101001111010101111
10001000011111110111
01001010101101110001
10101111111010000000
10001100010010111110
11000101110011000111
00101011011111101111
10000001101000110010
10110111111011111111
01101000101110101111
10011010111010100111
00010000000010000001
00001011011000100011
10001001100000101110
01110001110100001110
00001110100101001111
01111000100101001111
00100000100100101110
10111010011011000001
00001001101110111111
11111101111100101110
11110101010001011111
10100001001111111111
10101001011111110111
01000010111011111001
01110000010011110010
01001101000010110101
10011000111010100111
01010000000011000111
11100100101111110111
11111101111000010011
01000001101100111101
10001011001010111011
01011000111000100111
01011100001110000101
11101000010100001010
10100111110110110010
10011000111101100010
10011001001101100111
10111111011110100111
01110111111110010101
00110100111001111111
00000101100100000010
00000011011000001011
01000000010000010011
00010001001010101111
10101101001000000011
10010001011110101110
00100100101010110011
10001001111001101111
00010001001000011111
11101001000010010001
10001101011111101111
00101111110100101111
10011111111010110010
10110111110010101111
11010101110000101111
00001000111110101111
01100011101010100011
11110000101110100111
10000000000000100011
01001000000010000010
11010111100001110111
10000001000001001110
00000101001010110110
00100011000011110010
11111001001100000010
00111010110110111111
01000010001010000011
11111111011110101011
10000101001010000011
01100011100010110100
01110001001000100100
10110011101110110111
01100011001111111100
01110110101001111111
01110001111111100111
11011110111110001110
01010000101010011011
10111011101111011100
00110101001000110010
00010111011010101101
01001000011110111110
11111001101000010010
11111011001000000010
11000111101001000011
01101001001100101110
01011011001110111011
10010001101111111110
11111110111110010101
11110110111111001111
11000101111111111110
10111101101100010100
01111100001001111111
01010000000000010000
10111110111010010111
00100010101110011011
01111100101110101111
00000001101000010000
10100111001101100011
00011111111000111111
00011001001001111010
10100100011110110010
11011110111000011011
10111111100110111010
01111011010010001010
00100100011110011101
00100100111111111011
11011111101110110111
01110001011010000000
11010110101011111101
00000011111010100101
11100100111011111111
00100011111011101110
11110111111110110010
11000111010010000111
10010001101010101111
00100101101110111011
00001001100111101110
11100001011101111111
11001101001010000111
00111011111011000010
00010101011000001111
