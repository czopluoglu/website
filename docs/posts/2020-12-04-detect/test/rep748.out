-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep748.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.83
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7816
                   IDN index value:   0.7368
                           Ratio r:   0.6335

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    1    3    1    1    1    1    1
    4    4    4    4    4    4    4    4    5    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    4    6    7    8    9   10

   -----------CLUSTER  2-------------

    2

   -----------CLUSTER  3-------------

    5

   -----------CLUSTER  4-------------

   11   12   13   14   15   16   17   18   20

   -----------CLUSTER  5-------------

   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++++++++-+-+-----+
   +d++++++--+++--+----
   ++d++-+++++------+-+
   +++d++++--+----+-+++
   ++++d+++++--+-----+-
   ++-++d+-------+-----
   ++++++d-+----+------
   +++++--d-+--+---+---
   +-+-+-+-d--------++-
   +-+-+--+-d-+--------
   -+++------d+++-++-+-
   ++-------++d+-+++--+
   -+--+--+--++d-++-++-
   +-----+---+--d+-+++-
   -----+-----+++d+++++
   -+-+------+++-+d-+--
   -------+--++-++-d+-+
   --++----+---+++++d+-
   ---++---+-+-+++--+d+
   +-++-------+--+-+-+d
 
 No cross validation for this DETECT run
